autouri/gcsuri.py

```diff
@@ -138,30 +203,29 @@ def get_metadata(self, skip_md5=False, make_md5_file=False):
 
         try:
             b, _ = self.get_blob()
-            if b is not None:
-                # make keys lower-case
-                h = {k.lower(): v for k, v in b._properties.items()}
-                ex = True
-
-                if not skip_md5:
-                    if "md5hash" in h:
-                        md5 = parse_md5_str(h["md5hash"])
-                    elif "etag" in h:
-                        md5 = parse_md5_str(h["etag"])
-                    if md5 is None:
-                        # make_md5_file is ignored for GCSURI
-                        md5 = self.md5_from_file
-
-                if "size" in h:
-                    sz = int(h["size"])
-
-                if "updated" in h:
-                    mt = get_seconds_from_epoch(h["updated"])
-                elif "timecreated" in h:
-                    mt = get_seconds_from_epoch(h["timecreated"])
+            # make keys lower-case
+            h = {k.lower(): v for k, v in b._properties.items()}
```

Comment:
I think this would be more clear if we used longer variable names for `h`, `ex`, `mt` and `sz`.

Reply:
Renamed `h` to `headers` and `ex` to `exists`. Applied to other classes too.

Reply:
Excellent, I think this makes it way more readable.